CHANGELOG.md

```diff
@@ -1,42 +1,79 @@
- - Fixed Autofill on URL bar. ([#3012](https://github.com/brave/browser-laptop/issues/3012))
- - Fixed new tab button position fixes #3208. ([#3213](https://github.com/brave/browser-laptop/issues/3213))
- - Fixed Back/Forward navigation, tab shows page URL instead of page title. ([#3200](https://github.com/brave/browser-laptop/issues/3200))
- - Fixed tab title on back/forward fixes #3200. ([#3202](https://github.com/brave/browser-laptop/issues/3202))
- - Fixed Bookmarks Menu should list the bookmarks like other browsers do enhancement. ([#1993](https://github.com/brave/browser-laptop/issues/1993))
- - Fixed "auto open bookmarks toolbar for the first bookmark" test needs to be udpated. ([#2941](https://github.com/brave/browser-laptop/issues/2941))
- - Fixed "Pinning with partitions" test is failing but the functionality works. ([#2942](https://github.com/brave/browser-laptop/issues/2942))
- - Fixed Application icon is with poor resolution on Linux. ([#3229](https://github.com/brave/browser-laptop/issues/3229))
- - Fixed Change the size of the Linux icon fixes #3229. ([#3230](https://github.com/brave/browser-laptop/issues/3230))
- - Fixed Users can create bookmark folders with no name. ([#3188](https://github.com/brave/browser-laptop/issues/3188))
- - Fixed Validate bookmark name before creation Fixes #3188. ([#3190](https://github.com/brave/browser-laptop/issues/3190))
- - Fixed etags for data file downloading haven't been working. ([#3222](https://github.com/brave/browser-laptop/issues/3222))
- - Fixed Make targets.host UNIQUE, which auto-creates index. ([#3215](https://github.com/brave/browser-laptop/issues/3215))
- - Fixed Use noScript state per site instead of blocking scripts. ([#3205](https://github.com/brave/browser-laptop/issues/3205))
- - Fixed Non-script version of DuckDuckGo when blocking scripts is not loaded from location bar at first instance. ([#3189](https://github.com/brave/browser-laptop/issues/3189))
- - Fixed Market graph on wsj.com. ([#2102](https://github.com/brave/browser-laptop/issues/2102))
- - Fixed tab-specific notifications should be closed when tab is closed. ([#3169](https://github.com/brave/browser-laptop/issues/3169))
- - Fixed Brave can't handle URLs with whitespace in it. ([#3167](https://github.com/brave/browser-laptop/issues/3167))
- - Fixed Whitespace in url fixes #3167. ([#3168](https://github.com/brave/browser-laptop/issues/3168))
- - Fixed Flash on http://www.y8.com/games/superfighters. ([#3082](https://github.com/brave/browser-laptop/issues/3082))
- - Fixed Bookmarks now show under Bookmarks menu. ([#3055](https://github.com/brave/browser-laptop/issues/3055))
- - Fixed Menu should only be built once. ([#3022](https://github.com/brave/browser-laptop/issues/3022))
- - Update to https everywere 5.2.0. ([#2581](https://github.com/brave/browser-laptop/issues/2581))
+ ### Autofill
```

Comment:
also this doesn't really convey to me as a user that an autofill feature was added. Just that an alignment issue was fixed for it (etc). I don't think users will care about it since it's a sub-thing of the task itself.